auth/src/api/interfaces/hashing-handler.ts

```diff
@@ -8,10 +8,12 @@ export abstract class HashingHandler {
      * This function hashes a given value.
      *
      * @param value The value to hash.
+     * @param oldHash An optional string to submit an old used hash. This verifies the same salt
+     * and a same result for equal strings.
      *
      * @returns The hashed value.
      */
-    public abstract hash(value: string): string;
+    public abstract hash(value: string, oldHash?: string): string;
```

Comment:
Where do you need it? This would make `is-equals` obsolete since `hash("password", "pw-hash") == "pw-hash")` is essentially is-equals.